tests/unit/test_analytics.py

```diff
@@ -233,19 +233,6 @@ def test_run_sleep_detection(sleep_detection: analytics.GGIRSleepDetection) -> N
     assert isinstance(result, List)
 
 
```

Comment:
Why is this removed?

Reply:
I believe I've found the commit that made this change, and It's a merge from the orchestrator branch, 

https://github.com/childmindresearch/wristpy/pull/104/commits/7d54381f3baa616d975a7201e59b991182384ba4

Reply:
OH wait, sorry I confused myself, I just remembered the point of this. We added the model validator to the Settings, so we decided to get rid of all these other checks on the thresholds, as that's now covered by the validator.